### PR TITLE
DM-39214: Move ingredient pipelines into _ingredients dir

### PR DESCRIPTION
**Pull request by leeskelvin**

### CODE REVIEW

File: pipelines/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Pipeline Definitions
+
+This directory contains pipeline definition YAML files which are used when processing data with the LSST Science Pipelines.
+
+The pipelines defined here come in three flavors: camera-specific (within named directories), camera-agnostic (top-level, if any), and building-block ingredients (within the [\_ingredients](_ingredients) directory).
+Pipelines within the ingredients directory tend to be imported by other pipelines, and are not intended to be used directly by end-users for science purposes.
+
+The `pipetask build` command can be used to expand a pipeline YAML and resolve any imports for the purposes of visualizing it.
+For example, to visualize the `apPipe` subset from the [LsstCamImSim ApPipe pipeline](https://github.com/lsst/ap_pipe/blob/main/pipelines/LsstCamImSim/ApPipe.yaml) pipeline, run:
```
Comment (kfindeisen) [inline pipelines/README.md:9]: ImSim is the colloquial name of the instrument; also, these directory names are nonstandard and will be changed in [DM-36796](https://jira.lsstcorp.org/browse/DM-36796).
```suggestion
For example, to visualize the `apPipe` subset from the [ImSim ApPipe pipeline](https://github.com/lsst/ap_pipe/blob/main/pipelines/LsstCamImSim/ApPipe.yaml) pipeline, run:
```

**Review by kfindeisen — APPROVED** [review]
Looks good, just some documentation nitpicks.

File: pipelines/README.md (hunk 1 above)
Comment (leeskelvin) [reply pipelines/README.md:9]: Thanks for the pointer. In which case, I'd prefer switching to `LsstCam-imSim` for consistency with other uses elsewhere in the stack. Do you concur?

Comment (kfindeisen) [reply pipelines/README.md:9]: I think it sounds a bit odd -- I've never heard "I need some LSSTCam ImSim data" in a conversation...

Comment (leeskelvin) [reply pipelines/README.md:9]: This seems to be standard practice in other documentation:
* in obs_lsst: ["Metadata translation class for LSSTCam imSim headers"](https://pipelines.lsst.io/py-api/lsst.obs.lsst.translators.LsstCamImSimTranslator.html)
* in the DC2 guide: ["The instrument is called LSSTCam-imSim"](https://pipelines.lsst.io/v/daily/getting-started/dc2-guide.html)
* in technotes: ["the camera geometry currently implemented for LSSTCam-imSim"](https://rtn-027.lsst.io/)
* in the ap_verify datasets documentation: ["[ap_verify_ci_dc2 (two overlapping LSSTCam ImSim images)](https://github.com/lsst/ap_verify_ci_dc2/)"](https://pipelines.lsst.io/v/daily/modules/lsst.ap.verify/datasets.html)

Conversely, references to "imsim" alone tend to refer instead to the software suite:
* ["Images are simulated by the imSim software package"](https://dp0-2.lsst.io/data-products-dp0-2/data-processing.html)
* ["The DESC simulated the DC2 images using the imSim package."](https://dp0-2.lsst.io/data-products-dp0-2/)

"imsim" alone does appear sometimes to refer to the dataset, but it tends to be prefaced by something else, e.g.:
* ["colors recovered on simulated (DC2 ImSim) red cluster galaxies"](https://github.com/lsst/pipelines_lsst_io/blob/main/releases/v23_0_0.rst)

As above, I think consistently using `LSSTCam-imSim` in documentation will help end users connect the dots between the docs, the way these data are referenced in the butler, and the storage directories of pipeline YAMLs/etc in the stack.

Comment (kfindeisen) [reply pipelines/README.md:9]: Ok, I guess I was wrong about common usage. 🙂

**Pull request merged**